rocAL/rocAL_pybind/example/new_api/PYTHON_UNITTEST_TEST_FILE.sh

```diff
@@ -0,0 +1,63 @@
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name saturation --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.saturation.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name color_twist --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.color_twist.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name crop_mirror_normalize --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.crop_mirror_normalize.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name nop --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.nop.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name centre_crop --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.centre_crop.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name color_temp --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.color_temp.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name copy --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.copy.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name rotate_fisheye_fog --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.rotate_fisheye_fog.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name resize_brightness_jitter --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.resize_brightness_jitter.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name vignetter_blur --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 2>&1 | tee -a run.log.vignetter_blur.${CURRENTDATE}
```

Comment:
@swetha097 All values of tensor = 0. Not sure if this is the expected output for vignette_blur